app/code/community/Reverb/ReverbSync/etc/config.xml

```diff
@@ -333,6 +333,12 @@
         <order_sync_reverb_source_url>/api/my/orders/selling/all?created_start_date=%s</order_sync_reverb_source_url>
         <!-- Default orders to be synced to the Admin store -->
         <store_to_sync_order_to>0</store_to_sync_order_to>
+        <!-- The set of order statuses which should be regarded as constituting NOT being a paid order -->
+        <non_paid_order_statuses>
```

Comment:
Kyle and I talked this through and we agree. The logic should actually be "only create orders on update when status == paid". All other statuses should be updated but not used for order creation. @dunagan5887